### ISSUE TITLE
v1.3 of Fabric Essentials is not loading in MCreator

### ISSUE THREAD

**Issue by PSGitHubUser1** [NerdyPuzzle/Fabric-Essentials#2]
I am trying to load this plugin but its not loading.

Version: 1.3 [Latest]

MCreator: 2022.3

**Comment by NerdyPuzzle** [comment]
Go into more detail. I currently have the plugin loaded so I am positive that this issue is invalid.

**Comment by PSGitHubUser1** [comment]
> Go into more detail. I currently have the plugin loaded so I am positive that this issue is invalid.

Actually, I download the plugin and tried to import it but in UI of MCreator, it is not showing nothing like 'Fabric Essentials' or something. Let me try again. if it didn't worked then I will update this comment.

**Comment by NerdyPuzzle** [comment]
you need mcreator 2022.3 and the latest version of the fabric generator.

**Comment by NerdyPuzzle** [comment]
Seems I zipped the file wrong on accident. Updating the file now.

**Comment by PSGitHubUser1** [comment]
> you need mcreator 2022.3 and the latest version of the fabric generator.

I am on latest.

**Comment by PSGitHubUser1** [comment]
> Seems I zipped the file wrong on accident. Updating the file now.

Yea, Ok. Thanks. It happens.

And YAY! I again found a valid bug to report a dev.
